src/archana/Assi19_Character.java

```diff
@@ -0,0 +1,61 @@
+package archana;
+
+public class Assi19_Character {
+	char ch;
+	int count;
+	int uppercount;
+	int lowercount;
+
+	void totalLettersinstring(String str) {
+		count = 0;
```

Comment:
@arc700 you should write one method & inside that handle all these conditions with if-else ledders.